### PR TITLE
changed publicKey to verificationMethod in test vectors

### PR DESCRIPTION
**Pull request by acarnagey**
Re: https://github.com/w3c-ccg/did-method-key/issues/34
replaces publicKey in test vectors with verificationMethod

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -240,7 +240,7 @@ <h3>Create</h3>
 {
   "@context": "https://w3id.org/did/v1",
```
Comment (OR13) [inline index.html:241]: @acarnagey actually canyou fix this as well

Comment (OR13) [reply index.html:241]: use https://did.key.transmute.industries/ and `application/did+ld+json`... consider adding data for the `did+json` and JWK format as well.

**Review by OR13 — CHANGES_REQUESTED** [review]
please fix the context issue as well.

**Comment by decentralgabe** [comment]
thanks for this!

**Review by OR13 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**